README.md

```diff
@@ -89,3 +89,7 @@ spec:
 ```
 
 - Delete the `ibm-management-ingress-operator-xxxx` pod if you don't see any `management-ingress-xxx` pod in the `ibm-common-services` namespace
+
+## SecurityContextConstraints Requirements
+
+The IAM operator service does not support running under the OpenShift Container Platform default restricted security context constraints.
```

Comment:
`auth-idp`, `auth-pdp` and `auth-pap` won't run under the default restricted scc